src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_mgmt_commands.py

```diff
@@ -54,7 +54,8 @@ def test_batch_general_arm_cmd(self, resource_group, storage_account):
             self.check('encryption.keySource', 'Microsoft.Batch'),
             self.check('resourceGroup', '{rg}')])
 
-        time.sleep(100)
+        if self.is_live or self.in_recording:
```

Comment:
Why do we need to add this logic?

Reply:
To speed up the test in playback mode by only `sleep` in live or recording mode. It's a common pattern in our test code, I just find we forget to wrap it here so adding it here.